operatorcourier/flatten.py

```diff
@@ -0,0 +1,151 @@
+import logging
+import os
+from yaml import safe_load, MarkedYAMLError
+import semver
+import ntpath
```

Comment:
Please, no! This is a platform specific module for Windows. `os.path` will do the job.

> Since different operating systems have different path name conventions, there are several versions of this module in the standard library. The os.path module is always the path module suitable for the operating system Python is running on, and therefore usable for local paths.

Reply:
+1